### PR TITLE
dev-util/mdds: bump to 2.0.0, bump & fix revdeps

### PR DESCRIPTION
**Pull request by telans**
Closes: https://bugs.gentoo.org/713586
Signed-off-by: James Beddek <telans@posteo.de>

### CODE REVIEW

**Comment by telans** [comment]
Blocked until `app-office/libreoffice` makes use of the new ABI/API changes

**Comment by jospezial** [comment]
> Blocked until `app-office/libreoffice` makes use of the new ABI/API changes

There it is:
https://github.com/LibreOffice/core/commit/eb07a0e76fe240a184348d96a6cebf7c0a229ac0
"Upgrade mdds and liborcus to 2.0.0 and 0.17.0, respectively."

So it will be needed for app-office/libreoffice-9999 now.

**Comment by a17r** [comment]
It is rather easy. This can't have keywords until a LibreOffice release makes it relevant.

**Comment by jospezial** [comment]
> It is rather easy. This can't have keywords until a LibreOffice release makes it relevant.

Can we have at least the live ebuild changes in the tree? And maybe the release ebuilds unkeyworded bumped?
For people who use libreoffice-9999.

**Comment by jospezial** [comment]
app-office/libreoffice-9999.ebuild also needs adapted to these new slots.

**Comment by telans** [comment]
You should be able to use this PR now.

**Comment by telans** [comment]
`do not merge` can probably be removed now that all the new ebuilds are unkeyworded.

File: app-text/libetonyek/libetonyek-9999.ebuild [hunk 1]
```diff
@@ -45,6 +47,7 @@ src_prepare() {
 src_configure() {
 	local myeconfargs=(
 		--disable-werror
+		--with-mdds="${MDDS_VER}"
```
Comment (jospezial) [inline app-text/libetonyek/libetonyek-9999.ebuild:50]: @telans 
https://bugs.gentoo.org/show_bug.cgi?id=830462

**Pull request closed without merging**